src/diffusers/loaders/lora.py

```diff
@@ -191,6 +205,17 @@ def lora_state_dict(
                 guarantee the timeliness or safety of the source, and you should refer to the mirror site for more
                 information.
 
+        Example:
+
+        ```py
+        from diffusers import DiffusionPipeline
+        import torch
+
+        pipeline = DiffusionPipeline.from_pretrained(
+            "stabilityai/stable-diffusion-xl-base-1.0", torch_dtype=torch.float16
+        ).to("cuda")
+        pipeline.lora_state_dict("nerijs/pixel-art-xl")
```

Comment:
Nice example. Even though it's a public API, we don't advertise using it. We always recommend coming via the `load_lora_weights()` route and consider that to be the single source of entry.

Reply:
Removing then to avoid advertising it :)